<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoftwareStateFilter</class>
 <widget class="QWidget" name="SoftwareStateFilter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>234</width>
    <height>25</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="1">
    <widget class="QToolButton" name="toolButtonCorrect">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="toolTip">
      <string>Toggle software-state 'correct'</string>
     </property>
     <property name="statusTip">
      <string>Toggle software-state 'correct'</string>
     </property>
     <property name="icon">
      <iconset resource="qmc2.qrc">
       <normaloff>:/data/img/software_correct.png</normaloff>:/data/img/software_correct.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QToolButton" name="toolButtonUnknown">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="toolTip">
      <string>Toggle software-state 'unknown'</string>
     </property>
     <property name="statusTip">
      <string>Toggle software-state 'unknown'</string>
     </property>
     <property name="icon">
      <iconset resource="qmc2.qrc">
       <normaloff>:/data/img/software_unknown.png</normaloff>:/data/img/software_unknown.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="toolButtonMostlyCorrect">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="toolTip">
      <string>Toggle software-state 'mostly correct'</string>
     </property>
     <property name="statusTip">
      <string>Toggle software-state 'mostly correct'</string>
     </property>
     <property name="icon">
      <iconset resource="qmc2.qrc">
       <normaloff>:/data/img/software_mostlycorrect.png</normaloff>:/data/img/software_mostlycorrect.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QToolButton" name="toolButtonNotFound">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="toolTip">
      <string>Toggle software-state 'not found'</string>
     </property>
     <property name="statusTip">
      <string>Toggle software-state 'not found'</string>
     </property>
     <property name="icon">
      <iconset resource="qmc2.qrc">
       <normaloff>:/data/img/software_notfound.png</normaloff>:/data/img/software_notfound.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="toolButtonIncorrect">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="toolTip">
      <string>Toggle software-state 'incorrect'</string>
     </property>
     <property name="statusTip">
      <string>Toggle software-state 'incorrect'</string>
     </property>
     <property name="icon">
      <iconset resource="qmc2.qrc">
       <normaloff>:/data/img/software_incorrect.png</normaloff>:/data/img/software_incorrect.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="checkBoxStateFilter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="toolTip">
      <string>Toggle filtering by software-states</string>
     </property>
     <property name="statusTip">
      <string>Toggle filtering by software-states</string>
     </property>
     <property name="text">
      <string>State filter</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkBoxStateFilter</tabstop>
  <tabstop>toolButtonCorrect</tabstop>
  <tabstop>toolButtonMostlyCorrect</tabstop>
  <tabstop>toolButtonIncorrect</tabstop>
  <tabstop>toolButtonNotFound</tabstop>
  <tabstop>toolButtonUnknown</tabstop>
 </tabstops>
 <resources>
  <include location="qmc2.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>checkBoxStateFilter</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolButtonCorrect</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>13</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxStateFilter</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolButtonMostlyCorrect</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxStateFilter</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolButtonIncorrect</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>8</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxStateFilter</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolButtonNotFound</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>2</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>2</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxStateFilter</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolButtonUnknown</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>4</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
